templates/includes/install-prepare-adb.hbs

```diff
@@ -2,7 +2,7 @@
         <h3>Install ADB & Fastboot</h3>
         <br>
         <div name="LinAdb" style="display: block">
-          <h4>Debian / Ubuntu</h4>
+          <h4>Debian, Ubuntu, etc.</h4>
```

Comment:
Visually, i am not much a fan of unnecessarily repetitive elements like the `etc.` in multiple lines.
For this line, i'd suggest to keep the  `Debian / Ubuntu` since everyone using a derivative distro is very likely familiar with those names and the `etc.`  does not add much here . Also, this is supposed to be a headline and it looks unfortunate to end a headline on a `.` imo.